src/args.jl

```diff
@@ -1637,10 +1651,7 @@ function _update_series_attributes!(plotattributes::AKW, plt::Plot, sp::Subplot)
     end
 
     # set label
-    label = plotattributes[:label]
-    label = (label == "AUTO" ? "y$globalIndex" : label)
-    label = label in (:none, nothing, false) ? "" : label
-    plotattributes[:label] = label
+    plotattributes[:label] = label_to_string(plotattributes[:label], globalIndex)
```

Comment:
```suggestion
    plotattributes[:label] = label_to_string.(plotattributes[:label], globalIndex)
```
This should fix the tests.

Reply:
will it be okay to ask how? (because it was working even without broadcasting)